components/general/StandardButton.js

```diff
@@ -22,7 +22,7 @@ function StandardButton(
         onClick,
         disabled,
         children,
-        type = "primary",
+        type = "secondary",
```

Comment:
this changes all standard button on the site to default to secondary - is that correct?